[2024-12-06T20:30:29.589+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T20:30:29.604+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LocalIngestionDag_v02.ingest scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2024-12-06T20:30:29.621+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LocalIngestionDag_v02.ingest scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2024-12-06T20:30:29.622+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-06T20:30:29.920+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ingest> on 2021-01-02 06:00:00+00:00
[2024-12-06T20:30:29.933+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4162) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T20:30:29.939+0000] {standard_task_runner.py:72} INFO - Started process 4167 to run task
[2024-12-06T20:30:29.938+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag_v02', 'ingest', 'scheduled__2021-01-02T06:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpcixunx2o']
[2024-12-06T20:30:29.945+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask ingest
[2024-12-06T20:30:30.030+0000] {task_command.py:467} INFO - Running <TaskInstance: LocalIngestionDag_v02.ingest scheduled__2021-01-02T06:00:00+00:00 [running]> on host 39ecaeffba2c
[2024-12-06T20:30:30.160+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LocalIngestionDag_v02' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2021-01-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-01-02T06:00:00+00:00'
[2024-12-06T20:30:30.165+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T20:30:30.187+0000] {logging_mixin.py:190} INFO - root root pgdatabase 5432 ny_taxi yellow_taxi_2021_01 /opt/***/output_2021-01.parquet
[2024-12-06T20:30:30.204+0000] {logging_mixin.py:190} INFO - connection established, inserting data...
[2024-12-06T20:30:32.053+0000] {logging_mixin.py:190} INFO - inserted, took 1.848 second
[2024-12-06T20:30:32.061+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T20:30:32.069+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T20:30:32.069+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag_v02, task_id=ingest, run_id=scheduled__2021-01-02T06:00:00+00:00, execution_date=20210102T060000, start_date=20241206T203029, end_date=20241206T203032
[2024-12-06T20:30:32.118+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T20:30:32.140+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-06T20:30:32.141+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
